data-upgrade-news/src/main/java/org/exoplatform/news/upgrade/targets/PublishedNewsDisplayedPropUpgrade.java

```diff
@@ -67,61 +69,86 @@ public PublishedNewsDisplayedPropUpgrade(InitParams initParams,
     this.container = container;
   }
 
-  public int getPublishedNewsDisplayedPropCount() {
-    return publishedNewsDisplayedPropCount;
+  public int getMigratedPublishedNewsCount() {
+    return migratedPublishedNewsCount;
   }
 
   @Override
   public void processUpgrade(String oldVersion, String newVersion) {
     long startupTime = System.currentTimeMillis();
-    LOG.info("Start upgrade of published news displayed property");
-
-    boolean transactionStarted = false;
-    RequestLifeCycle.begin(container);
-    EntityManager entityManager = entityManagerService.getEntityManager();
+    LOG.info("Start published news migration");
+    List<MetadataItemEntity> newsTargetsMetadataItems = new ArrayList<>();
```

Comment:
```suggestion
    List<MetadataItemEntity> newsTargetsMetadataItems = null;
```
no need to initialize this variable IMO